src/index.ts

```diff
@@ -67,6 +70,10 @@ async function runUserScripts(entrypoint: string) {
   }
 }
 
+function isExecaError(err: Error): err is ExecaError {
```

Comment:
Have to roll out our own type guard to make sure we are dealing with an `ExecaError`

Reply:
Thank you!